pkg/diskboot/device.go

```diff
@@ -24,6 +24,31 @@ type Device struct {
 
 // fstypes returns all block file system supported by the linuxboot kernel
 
+// FindDevicesRW searches for devices with bootable configs and RW mount.
+func FindDevicesRW(devicesGlob string) (devices []*Device) {
+	fstypes, err := storage.GetSupportedFilesystems()
+	if err != nil {
+		return nil
+	}
+
+	sysList, err := filepath.Glob(devicesGlob)
+	if err != nil {
+		return nil
+	}
+	// The Linux /sys file system is a bit, er, awkward. You can't find
+	// the device special in there; just everything else.
```

Comment:
I may be missing something but

rminnich@xcpu:~/gopath/src/github.com/u-root/cpu/cmds/cpu$ cat /sys/block/sda/dev
8:0
rminnich@xcpu:~/gopath/src/github.com/u-root/cpu/cmds/cpu$

Reply:
Its a copy paste of the function below, its a temporary copy paste till we modify FindDevice and all its callers to pass flags as additional parameter..